### PR TITLE
Add more detail to release playbook

### PR DESCRIPTION
**Pull request by keertk**

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -12,16 +12,23 @@ more details about how the process works, see
...
 3. Identify and set the following environment variables:
 
   * `COMMIT_HASH` the commit hash where the pipeline was run (see below)
-  * `NEW_VERSION` the new version number you’re trying to release (e.g. `3.1`)
+  * `NEW_VERSION` the new version number you’re trying to release (e.g. `11.09`)
   * `RC` the number of the current release candidate
 
-4. Create a new release candidate by running the command below from the bazel repo:
+     For example:
+     ```
```
Comment (comius) [inline docs/release.md:25]: nit: I think you can add "bash" after the triple back quote for nicer rendering:
~~~
```bash
export ...
```
~~~

same in the block below

File: docs/release.md [hunk 2]
```diff
@@ -12,16 +12,23 @@ more details about how the process works, see
...
+     EXPORT COMMIT_HASH=7bd0ab63a8441c3f3d7f495d09ed2bed38762874
```
[the remark this answers is not in the document]
Comment (keertk) [reply docs/release.md]: Done

File: docs/release.md [hunk 3]
```diff
@@ -32,27 +39,119 @@ an example.
...
-5. Create a new bazel Pull Request that updates the `java_tools` archives (file 
-[distdir_deps.bzl](https://github.com/bazelbuild/bazel/blob/master/distdir_deps.bzl))
-with the new release candidates.
-The PR triggers the CI presubmit.
+5. Create a new bazel Pull Request that updates the `java_tools` archives (file [distdir_deps.bzl](https://github.com/bazelbuild/bazel/blob/master/distdir_deps.bzl)) with the new release candidates. The PR triggers the CI presubmit.  
 
+  * Edit [distdir_deps.bzl](https://github.com/bazelbuild/bazel/blob/master/distdir_deps.bzl) by updating the `archive`, `sha256`, and `urls` fields for `remote_java_tools` with the correct version, rc, sha256sum, and url (see output from step 4)
+  
+     Example:
+     ```
```
[the remark this answers is not in the document]
Comment (keertk) [reply docs/release.md]: Done

File: docs/release.md (hunk 1 above)
Comment (keertk) [reply docs/release.md]: Done

**Review by comius — APPROVED** [review]

**Pull request merged**